tests/e2e/pipeline.go

```diff
@@ -110,15 +110,16 @@ func verifyPipelinePaused(namespace string, pipelineRolloutName string, pipeline
 	}, testTimeout).Should(Equal(metav1.ConditionTrue))
 
 	document("Verify that Pipeline is paused and fully drained")
-	verifyPipelineStatusEventually(Namespace, pipelineName,
+	verifyPipelineStatusEventually(namespace, pipelineName,
 		func(retrievedPipelineSpec numaflowv1.PipelineSpec, retrievedPipelineStatus numaflowv1.PipelineStatus) bool {
 			return retrievedPipelineStatus.Phase == numaflowv1.PipelinePhasePaused && retrievedPipelineStatus.DrainedOnPause
 
 		})
-	verifyPodsRunning(namespace, 0, getVertexLabelSelector(pipelineName))
+	// this happens too fast to verify it:
+	//verifyPodsRunning(namespace, 0, getVertexLabelSelector(pipelineName))
```

Comment:
this can happen too fast now to verify 0 Pods now but note that Numaflow has in fact indicated phase=Paused and drainedOnPause=true in the previous step, which it would only do if there were 0 Pods